Mathlib/Data/Nat/ModEq.lean

```diff
@@ -412,6 +412,11 @@ theorem le_mod_add_mod_of_dvd_add_of_not_dvd {a b c : ℕ} (h : c ∣ a + b) (ha
     have : (a + b) % c = a % c + b % c := add_mod_of_add_mod_lt (lt_of_not_ge hc)
     simp_all [dvd_iff_mod_eq_zero]
 
+lemma mod_sub_comm {a b n : ℕ} (h : b ≤ a % n) : a % n - b = (a - b) % n := by
```

Comment:
Maybe a better name would be `mod_sub_of_le`?

Reply:
All done.